src/auro-toast.js

```diff
@@ -78,33 +99,85 @@ export class AuroToast extends LitElement {
    * @returns {void}
    */
   fadeOutToast() {
-    const toastContainer = this.shadowRoot.querySelector('#toastContainer');
-    toastContainer.className = 'hidden';
+    const toastContainer = this.shadowRoot.querySelector('.toastContainer');
+    toastContainer.classList.add('hidden');
 
-    // Wait until fade out completes to set visible to false
     setTimeout(() => {
-      this.visible = false;
+      this.closeToast();
     }, FADE_OUT_DURATION);
   }
 
-  updated() {
-    if (this.visible) {
+  /**
+   * @private
+   * @returns {void}
+   */
+  closeToast() {
+    clearTimeout(this.fadeOutTimer);
+    this.visible = false;
+    this.dispatchEvent(new CustomEvent('onToastClose', {
+      bubbles: true,
+      composed: true,
+      detail: this
+    }));
+  }
+
+  /**
+   * For mobile, set the onclick function so the toast can be dismissed if it is tapped on anywhere inside the toast.
+   * @private
+   * @returns {void}
+   */
+  setOnClick() {
+    const mobileBreakPoint = 767;
+    if (window.innerWidth < mobileBreakPoint) {
+      this.onclick = () => {
+        this.fadeOutToast();
+      };
+    }
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    this.setOnClick();
+  }
+
+  updated(changedProperties) {
+    if (changedProperties.has('variant')) {
+      clearTimeout(this.fadeOutTimer);
+    }
+    // do not auto dismiss for error toasts
+    if (this.visible && this.variant !== 'error') {
       this.fadeOutTimer = setTimeout(() => {
         this.fadeOutToast();
       }, TIME_TIL_FADE_OUT);
     }
   }
 
   render() {
-    return this.visible
-      ? html`
-      <div id="toastContainer">
-        <slot></slot>
-        <button class="toastButton" @click="${this.handleOnClose}">
-          ${this.svg}
-        </button>
-      </div>
-      `
+    let iconHtml = html``;
+
+    switch (this.variant) {
+      case undefined:
+      case null:
+        // default toast uses information icon
+        iconHtml = this.generateIconHtml(information.svg);
+        break;
+      case "error":
+        iconHtml = this.generateIconHtml(error.svg);
+        break;
+      case "success":
+        iconHtml = this.generateIconHtml(success.svg);
+        break;
+      default:
+        break;
+    }
+
+    return this.visible ? html`<div class="toastContainer">
```

Comment:
```suggestion
    return this.visible ? html`<div class="toastContainer" aria-live="polite">
```

Reply:
resolved in this commit: https://github.com/AlaskaAirlines/auro-toast/pull/20/commits/9815e230ab260d724f4804ae3d84368f487db059